Scripts/Player.cs

```diff
@@ -161,6 +162,12 @@ public void Move(Vector2 weightedDirection, float delta)
         }
     }
 
+    public void WeaponFired(Projectile projectile) 
+    {
+        GD.Print(projectile.Velocity.Length());
+        ApplyCentralImpulse(-10 * projectile.Mass * projectile.Velocity);

```

Comment:
What is 10?

Reply:
Some multiplier to make it look more obvious lol

Reply:
Lol ok

Reply:
Zis